ui_components/src/profile/Profile.css

```diff
@@ -0,0 +1,94 @@
+
+.profile_container {
+    max-width: 100%;
+
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    padding: 20px;
+    box-sizing: border-box;
+}
+
+.profile_avatar_container {
+    width: 230px;
+    height: 250px;
+    margin-right: 20px;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.status_attributes {
+    min-width: 600px;
+    flex-grow: 1;
+    min-height: 210px;
+}
+
+/* Stats */
+.stats_container {
+    width: 600px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 8px;
+}
+.total_stats {
+    width: 100%;
+    box-sizing: border-box;
+    padding: 5px 10px;
+}
+
+.stat_list {
+    width: 47%;
+
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
+.stat_list.attributes {
+    width: 50%;
+}
+
+.stat {
+    width: 100%;
+    height: 84px;
+    position: relative;
+
+    font-size: 12px;
```

Comment:
Can we use `var(--font-size)` variables for this? Will make it easier if we want to adjust font size between layouts in the future or allow for global layout settings (e.g. dropdown -> font size: Large).

Reply:
Yup I'll update that 👍